entities/entities/food/init.lua

```diff
@@ -45,7 +45,7 @@ function ENT:Use(activator, caller)
     umsg.End()
 
     self:Remove()
-    activator:EmitSound("vo/sandwicheat09.mp3", 100, 100)
+    activator:EmitSound(self.UseSound, 100, 100)
```

Comment:
How does `EatSound` turn into `UseSound`?

Reply:
Didn't i fix this within a few hours?
I'm running this on my server for days now.
No problems.

Reply:
Ah sorry, just seen it.
I wasn't sure how to name it.

And the food entity isn't used that much on our server, so I wouldn't know.

Fixed it, and again, very sorry.
I hate doing these mistakes.

Reply:
Testing is best done by ensuring the code path is run and watching  the results. In this case it would be spawning some food and trying to eat it, see if it makes a sound.